resources/metadata/tmsUpload.yaml

```diff
@@ -78,7 +78,7 @@ spec:
           - STEPS
           - STAGES
       - name: nodeExtDescriptorMapping
-        type: map[string]interface{}
+        type: string
```

Comment:
Could you add more context to this PR to why this needs to change the type?

Reply:
the change was reverted